### DISCUSSION TITLE
Adventure map, Hut of the Magi, allow to interrupt the automatic sequence that shows the areas of the adventure map revealed by the Eyes of the Magi [Done]

### DISCUSSION THREAD

**Discussion by LeHerosInconnu** [ihhub/fheroes2#7105] [General]
When the Hut of the Magi is visited, an automatic sequence is initiated that moves the adventure map view to each Eye of the Magi to show the area revealed by each Eye of the Magi.
This is an improvement over the original game.
However, as soon as the number of Eye of the Magi is higher than say three, it becomes really annoying because it takes time.
I suggest that the user can interrupt this automatic sequence by pressing any mouse button or keyboard key.
The areas revealed by each Eye of the Magi are then revealed intantaneously.

Here is the scenario test file:
[0 9953a10.zip](https://github.com/ihhub/fheroes2/files/11397997/0.9953a10.zip)

**Comment by Districh-ru** [comment]
Hi, @LeHerosInconnu. Do you mean interrupting the entire sequence or a mouse click should interrupt showing the current Eye of the Magi and switch to the next one without waiting?

**Reply by LeHerosInconnu** [reply]
Hello @Districh-ru,

> Hi, @LeHerosInconnu. Do you mean interrupting the entire sequence or a mouse click should interrupt showing the current each Eye of the Magi and switch to the next one without waiting?

A single action by the player ends the sequence completely and all areas are revealed instantly, and the player can continue playing.

**Comment by LeHerosInconnu** [comment]
This has been implemented.
I close the discussion.
